.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+on:
+  push:
+    branches:
+      - master
+      - branch-*
+  pull_request:
```

Comment:
Why do we need to trigger on `pull_request`? once we merge one it will go under the role above

Reply:
Dropped the lines